src/locale/locales/nl/messages.po

```diff
@@ -0,0 +1,7903 @@
+msgid "Change handle"
+msgstr "Wijzig handle"
+
+#: src/view/com/modals/VerifyEmail.tsx:155
+msgid "Change my email"
+msgstr "Wijzig mijn e-mailadres"
+
+#: src/view/screens/Settings/index.tsx:730
+msgid "Change password"
+msgstr "Wijzig wachtwoord"
+
+#: src/view/com/composer/select-language/SuggestedLanguage.tsx:73
+msgid "Change post language to {0}"
+msgstr "Wijzig berichttaal naar {0}"
+
+#: src/Navigation.tsx:337
+#: src/view/shell/bottom-bar/BottomBar.tsx:200
+#: src/view/shell/desktop/LeftNav.tsx:329
+#: src/view/shell/Drawer.tsx:446
+msgid "Chat"
+msgstr "Chat"
+
+#: src/screens/Messages/Settings.tsx:61
+#: src/view/screens/Settings/index.tsx:614
+msgid "Chat Settings"
+msgstr "Chatinstellingen"
+
+#: src/components/dms/ConvoMenu.tsx:82
+msgid "Chat muted"
+msgstr "Chat genegeerd"
+
+#: src/components/dms/ConvoMenu.tsx:112
+#: src/components/dms/MessageMenu.tsx:81
+#: src/Navigation.tsx:342
+#: src/screens/Messages/ChatList.tsx:88
+#: src/view/screens/Settings/index.tsx:605
+msgid "Chat settings"
+msgstr "Chatinstellingen"
+
+#: src/components/dms/ConvoMenu.tsx:84
+msgid "Chat unmuted"
+msgstr "Chat niet meer genegeerd"
+
+#: src/screens/SignupQueued.tsx:78
+#: src/screens/SignupQueued.tsx:82
+msgid "Check my status"
+msgstr "Controleer mijn status"
+
+#: src/screens/Login/LoginForm.tsx:275
+msgid "Check your email for a login code and enter it here."
+msgstr "Controleer uw e-mail voor een aanmeldcode en voer deze hier in."
+
+#: src/view/com/modals/DeleteAccount.tsx:231
+msgid "Check your inbox for an email with the confirmation code to enter below:"
+msgstr "Controleer uw inbox voor een e-mail met de bevestigingscode om hieronder in te voeren:"
+
+#: src/screens/StarterPack/Wizard/index.tsx:199
+msgid "Choose Feeds"
+msgstr "Kies Feeds"
+
+#: src/screens/StarterPack/Wizard/index.tsx:195
+msgid "Choose People"
+msgstr "Kies mensen"
+
+#: src/view/com/auth/server-input/index.tsx:76
+msgid "Choose Service"
+msgstr "Kies dienst"
+
+#: src/components/StarterPack/ProfileStarterPacks.tsx:291
+msgid "Choose for me"
+msgstr "Kies voor mij"
+
+#: src/view/com/composer/labels/LabelsBtn.tsx:116
+msgid "Choose self-labels that are applicable for the media you are posting. If none are selected, this post is suitable for all audiences."
+msgstr "Kies labels die van toepassing zijn op de media die u plaatst. Dit bericht is geschikt voor alle doelgroepen als er geen zijn geselecteerd."
+
+#: src/screens/Onboarding/StepFinished.tsx:271
+msgid "Choose the algorithms that power your custom feeds."
+msgstr "Kies de algoritmen die uw aangepaste feeds aansturen."
+
+#: src/screens/Onboarding/StepProfile/AvatarCreatorItems.tsx:107
+msgid "Choose this color as your avatar"
+msgstr "Kies deze kleur als uw avatar"
+
+#: src/screens/Signup/StepInfo/index.tsx:201
+msgid "Choose your password"
+msgstr "Kies uw wachtwoord"
+
+#: src/view/screens/Settings/index.tsx:877
+msgid "Clear all storage data"
+msgstr "Wis alle opslaggegevens"
```

Comment:
```suggestion
msgstr "Wis alle opslagen gegevens"
```

opgslaggegegevens can mean "data about storage" not "stored data"

Reply:
Thx for context. Changed in 3 places to "opgeslagen gegevens".